fastestimator/network.py

```diff
@@ -181,3 +174,62 @@ def _reduce_loss(self, batch, global_batch_size, epoch_losses, warm_up):
                     "please make sure loss is element-wise loss"
             reduced_loss[loss_name] = tf.reduce_sum(element_wise_loss) / global_batch_size
         return reduced_loss
+
+
+def build(model_def, model_name, optimizer, loss_name):
+    """build keras model instance in FastEstimator
+
+    Args:
+        model_def (function): function definition of tf.keras model
+        model_name (str, list, tuple): model name(s)
+        optimizer (str, optimizer, list, tuple): optimizer(s)
+        loss_name (str, list, tuple): loss name(s)
+
+    Returns:
+        model: model(s) compiled by FastEstimator
+    """
+    convert_tuple = False
```

Comment:
instead of going to all the hassle to maintain the tuple vs list distinction, lets just call to_list on everything right up front

Reply:
this also lets us get rid of the if-branching at line 198

Reply:
what we need is to return the same data structure that user defines in the model_def

Reply:
fixed